arduino-ide-extension/src/browser/contributions/survey-notification.ts

```diff
@@ -0,0 +1,62 @@
+import { MessageService } from '@theia/core';
+import { FrontendApplicationContribution } from '@theia/core/lib/browser';
+import { inject, injectable } from '@theia/core/shared/inversify';
+import { shell } from '@theia/electron/shared/electron';
```

Comment:
I am doing my best to get rid of the electron remote ASAP, please do not add the code. See my proposal below on how to open an external URL in Theia.

Reply:
I didn't know that, thanks. I'm going to use `WindowService` to open an external URL.